### DISCUSSION TITLE
What will happen when I put code in a media folder?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#481] [Ask a question] [answered]
Accidentally, I found the redirection of »Core…Location« entries of system.ini. 

What is the difference to »Core…Dir« ?

Is it possible, to address a php file with this redirection?

~~~
$TheFile = $this->yellow->system->get("serverBase").$this->yellow->system->get("ressourceLocation")."test.php";
~~~

Because the resources folder this will not work, placing "test.php" into extensions does not work either, if it is no extension (and in this case doesn't need any redirecting).

What will happen, if such a »test.php« is truly placed in a created location `/media/extensions/` ?

I was thinking about organizing additional helper files for extensions in `/media/extension/` but I want to avoid conflicts with this virtual path(es).

**Comment by markseuffert** [comment, marked as answer]
I answered you in datenstrom/community#471, in short don't do it. Source code in a media folder is [undefined behaviour](https://en.wikipedia.org/wiki/Undefined_behavior).
